### ISSUE TITLE
No compatible binary build found

### ISSUE THREAD

**Issue by jamgold** [jankapunkt/meteor-blaze-bs4#27]
**Describe the bug**
I have been testing your package in a project, and when I did `meteor update` I got the error message

```While checking for jkuester:blaze-bs4@0.15.3:
error: No compatible binary build found for this package. Contact the package author and ask them to publish it for your platform.
```

**To Reproduce**
1. Run `meteor update` or `meteor add jkuester:blaze-bs4`

**Expected behavior**
I expected `meteor update` to work how it always has

**Desktop (please complete the following information):**
 - OS: MacOS X 10.14.6
 - Browser chrome
 - Version 78.0.3904.70

**Additional context**

**Comment by jankapunkt** [comment]
Hi @jamgold, I will check on that at work today. There I have also an OSX and try to reproduce.

**Comment by jankapunkt** [comment]
I published 0.15.3 for arch osx, please check if it runs on your machine now.

**Comment by jamgold** [comment]
Thank you, it works now
